pypolygames/params.py

```diff
@@ -707,11 +716,11 @@ def __post_init__(self) -> None:
                 "In '--real_time' the evaluation follow the training "
                 "so '--checkpoint' should not be set"
             )
-        if self.checkpoint_dir is None and self.checkpoint is None:
-            raise ValueError(
-                "Either a '--checkpoint_dir' or a path to a '--checkpoint' "
-                "must be specified"
-            )
+        # if self.checkpoint_dir is None and self.checkpoint is None:
```

Comment:
Why is this check removed?

Reply:
This was necessary because otherwise if we do pure MCTS vs pure MCTS, it sees that there is no checkpoint dir and no checkpoint, and doesn't run (no checkpoint is necessary, since we're doing pure MCTS).

Reply:
You should probably change it to `if (pure_mcts) and (self.checkpoint_dir is None and self.checkpoint is None)`

Reply:
The challenge here is that pure_mcts is a ModelParam, but here we're in the EvalParam section. Ideas how to make that work?